lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -23,12 +23,11 @@ def self.run(params)
         params.values # to validate all inputs before looking for the ipa/apk
         auth_token = fetch_auth_token(params[:service_credentials_file])
         fad_api_client = Client::FirebaseAppDistributionApiClient.new(auth_token)
-        platform = Actions.lane_context[Actions::SharedValues::PLATFORM_NAME]
         binary_path = params[:ipa_path] || params[:apk_path]
 
         if params[:app] # Set app_id if it is specified as a parameter
           app_id = params[:app]
-        elsif platform == :ios
+        elsif @platform == :ios
```

Comment:
Prefer using  the `platform` method throughout this file instead of using the `@platform` variable directly.